www/js/domoticz.js

```diff
@@ -6282,6 +6282,89 @@ function fromInstanceOrFunction(functionTemplate = f => f()) {
 /* LiveSearch Functions: Filters devices when typing in the INPUT field ##################################### */
 var _debug_livesearch= false;
 
+function AddToLiveSearch(current_data, new_value) {
+	if (
+		(typeof new_value == 'undefined') ||
+		(new_value === "")
+	   ) {
+		return current_data;
+	}
+	if (
+		(typeof current_data == 'undefined') ||
+		(current_data === "")
+	) {
+		return new_value;
+	}
+	if (current_data.includes(new_value))
+		return current_data;
+	return current_data + " " +  new_value;
+}
+
+GenerateLiveSearchTextDefault = function (item) {
+	var searchText = "";
+	searchText = AddToLiveSearch(searchText, item.idx);
+	searchText = AddToLiveSearch(searchText, item.Name);
+	searchText = AddToLiveSearch(searchText, item.Description.replace('"',"'"));
+	searchText = AddToLiveSearch(searchText, item.Type);
```

Comment:
AddToLiveSearch(searchText, item.Type) breaks the ability to correctly search in Light/Switch Status, (because for example, it adds the "Switch On/Off" to the search text), thus preventing to find Switch by status.

Reply:
There is no "Switch On/Off" type, code here is correct